src/main/app/form/validation.ts

```diff
@@ -1,6 +1,8 @@
-import { CaseDate } from '../case/case';
+import * as EmailValidator from 'email-validator';
```

Comment:
Maybe this could be:
```suggestion
import { validate as validateEmail } from 'email-validator';
```